tools/indexer/wallet/src/main.rs

```diff
@@ -0,0 +1,221 @@
+use std::env;
+
+use bigdecimal::BigDecimal;
+use num_traits::cast::FromPrimitive;
+use tokio::sync::mpsc;
+
+use diesel::{
+    prelude::*,
+    r2d2::{ConnectionManager, Pool},
+};
+#[macro_use]
+extern crate diesel;
```

Comment:
this is old rust syntax. We don't need `extern crate` in most cases now.

Reply:
And what's the modern syntax? I took this lines from the official diesel guide. Can't find anything else.

Reply:
@bowenwang1996 diesel still did not get upgraded to support Rust 2018 imports for their macros. There were a few blockers there when I checked it the last time, so there is no other way around at the moment.